doc-content/drools-docs/src/main/asciidoc/DecisionEngine/performance-tuning-decision-engine-ref.adoc

```diff
@@ -36,3 +36,15 @@ try {
 
 For information about built-in event listeners and debug logging in the {DECISION_ENGINE}, see xref:engine-event-listeners-con_decision-engine[].
 --
+
+Use lambda externalization for executable model::
+Enable lambda externalization which optimizes the memory consumption at runtime. It rewrites lambdas that are generated and used in the executable model, enabling you to reuse the same lambda multiple times with all the patterns and the same constraint. The executable model becomes garbage collectible once the rete and phreak is instantiated.
++
+--
+Use the following property to enable lambda externalization for the executable model:
```

Comment:
s/Use the following property to enable lambda externalization for the executable model:/To enable lambda externalization for the executable model, include the following property:

Reply:
Done

Reply:
@csherrar can you please review the updates and provide your approval.

Reply:
Looks good to me @hmanwani-rh .